cmd/bad-key-revoker/main.go

```diff
@@ -340,6 +375,9 @@ func (bkr *badKeyRevoker) invoke() (bool, error) {
 	if err != nil {
 		return false, err
 	}
+
+	// Decrement 1 from the number of rows to be processed.
+	keysToProcess.Dec()
```

Comment:
I think we should avoid this, so we are always measuring the same thin in the same place. It's true that we know a little bit more about the number of rows processed now - we know that one has been removed. But we don't know anything about how many new rows have been added while we're working. We will get a good measurement next time we go around the loop, though.